src/libraries/Microsoft.NET.WebAssembly.Threading/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <!-- Disable all binplacing since this assembly will clash with the real ref assemblies -->
+    <EnableBinPlacing>false</EnableBinPlacing>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
```

Comment:
You shouldn't need any of these properties as projects in this directory don't compile. I believe you can believe this entire file.